core/metacling/src/TCling.cxx

```diff
@@ -6112,16 +6112,22 @@ static bool LookupBloomFilter(llvm::object::ObjectFile *soFile, uint32_t hash) {
    return  (bitmask & word) == bitmask;
 }
 
-// Lookup for normal symbols
-static bool LookupNormalSymbols(llvm::object::ObjectFile *RealSoFile, const std::string& mangled_name, const std::string& LibName)
+/// Looks up symbols from a an object file, representing the library.
+///\returns true on success.
+static bool LookupNormalSymbols(const std::string &library_filename, std::string mangled_name)
```

Comment:
Doesn't it look up just one symbol? Why plural?

Reply:
Legacy. I do not know what's `normal` too :). I've removed the plural at least.